### DISCUSSION TITLE
GitHub Universe: Super Early Bird Tickets Ending Soon!

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#65213] [Announcements]
[<img width="1200" alt="1200x300-Blog_Ad-Master@2x" src="https://github.com/community/community/assets/75342807/2cda67ca-12bb-41e2-b57f-194d1851b3f7">](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)

## GitHub Universe: Super Early Bird Tickets Ending Soon 🎟️ 
Early bird tickets are available until September 1st, 2023. Don't delay and secure your spot now! [[Get your Early-Bird tickets here](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)]

#### So what is GitHub Universe? 
- GitHub Universe is our annual global event that brings developers, enterprise leaders, and security professionals together for two days of networking, conversing with peers and the team at GitHub, and learning about the latest innovations. 
- This year, we’ll have over 80 sessions with 1,000 speakers covering AI, security, and DevEx.

Learn more about this annual event in our [Get to know GitHub Universe!](https://github.com/orgs/community/discussions/63371) post in the [Announcements category](https://github.com/orgs/community/discussions/categories/announcements)

[![Registration Screenshot](https://github.com/community/community/assets/75342807/cb24af4b-4f59-47e3-bea3-308c2d2a8eaf)](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)

**Comment by mihaiepure1977** [comment]
Cum pot sa imi recuperez fondue de active

**Comment by AndyReed36** [comment]
> [](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)<img alt="1200x300-Blog_Ad-Master@2x" width="1200" src="https://github.com/community/community/assets/75342807/2cda67ca-12bb-41e2-b57f-194d1851b3f7">
> 
> ## GitHub Universe: Super Early Bird Tickets Ending Soon 🎟️
> Early bird tickets are available until September 1st, 2023. Don't delay and secure your spot now! [[Get your Early-Bird tickets here](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)]
> 
> #### So what is GitHub Universe?
> * GitHub Universe is our annual global event that brings developers, enterprise leaders, and security professionals together for two days of networking, conversing with peers and the team at GitHub, and learning about the latest innovations.
> * This year, we’ll have over 80 sessions with 1,000 speakers covering AI, security, and DevEx.
> 
> Learn more about this annual event in our [Get to know GitHub Universe!](https://github.com/orgs/community/discussions/63371) post in the [Announcements category](https://github.com/orgs/community/discussions/categories/announcements)
> 
> [](https://githubuniverse.com/?utm_source=discussion&utm_medium=github&utm_campaign=launch)![Registration Screenshot](https://github.com/community/community/assets/75342807/cb24af4b-4f59-47e3-bea3-308c2d2a8eaf)
> 
> ### 🔗 Follow up from our [Universe 2023 tickets are available now! 🎟️ ](https://github.com/orgs/community/discussions/63834)

@ettaboyle I think there might be a typo there in the first line "_Early bird tickets are available until September 1st, 2023_".  I believe the Early bird tickets are available from Sep 1st-30th, yes?

**Comment by DMYTRO140787** [comment]
26132437

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
